internal/provider/utils.go

```diff
@@ -134,3 +134,13 @@ func formatEnumMarkdownList[T ~string](allowedValues []T) (mdList string) {
 	}
 	return mdList
 }
+
+type Knowable interface {
+	IsUnknown() bool
+	IsNull() bool
+}
+
+// IsKnown is a shortcut that checks in a value is neither null nor unknown.
```

Comment:
super duper typo nit:
```suggestion
// IsKnown is a shortcut that checks if a value is neither null nor unknown.
```

Reply:
Fixed!